TotalSegmentator/TotalSegmentator.py

```diff
@@ -298,7 +298,7 @@ def __init__(self):
 
         ScriptedLoadableModuleLogic.__init__(self)
 
-        self.totalSegmentatorPythonPackageDownloadUrl = "https://github.com/wasserth/TotalSegmentator/archive/b38eb449ad8652a987878a925203cbfa354e9b85.zip"  # tag: 1.5.3
+        self.totalSegmentatorPythonPackageDownloadUrl = "https://github.com/wasserth/TotalSegmentator/archive/b38eb449ad8652a987878a925203cbfa354e9b85.zip" #  version 1.5.3 
```

Comment:
Please keep the original line.

Python coding style requires double-space before the `#` in inline comments (and not after).
The git term for `1.5.3` is a `tag` (not `version`).